syft/pkg/cataloger/cpp/package.go

```diff
@@ -67,35 +67,43 @@ func splitConanRef(ref string) *conanRef {
 }
 
 func newConanfilePackage(m pkg.ConanfileEntry, locations ...file.Location) *pkg.Package {
-	return newConanPackage(m.Ref, m, locations...)
+	// though a conanfile is a listing of direct dependencies, we are not capturing these today, so are forced to answer incomplete
```

Comment:
https://github.com/anchore/syft/issues/572 and this file reverse conaninfo.txt and conanfile.txt. I'm not sure which one is right.

Reply:
572 is wrong, the code and table in PR description is right. I've updated the PR description on 572 to reflect the correct state + PR references